api/src/main/java/com/google/sps/Application.java

```diff
@@ -10,9 +10,9 @@
 
 public class Application {
 	public static void main(String[] args) {
-		SpringApplication.run(Application.class, args);
+    SpringApplication.run(Application.class, args);
 
-		try {
+    try {
```

Comment:
fix indentation?

Reply:
I thought I turned from tabs to spaces, I am not sure why it still has that problem. Going to look into fixing that